plugins/scaffolder/alpha-api-report.md

```diff
@@ -16,6 +20,14 @@ import { ScaffolderTaskOutput } from '@backstage/plugin-scaffolder-react';
 import { SubRouteRef } from '@backstage/core-plugin-api';
 import { TemplateEntityV1beta3 } from '@backstage/plugin-scaffolder-common';
 
+// @alpha
+export const CategoryPicker: () => JSX.Element | null;
```

Comment:
Thinking that these components should be moved to `-react` instead and be exported from there. Doesn't make sense for them to live in the plugin anymore.

Reply:
Also rename?

Reply:
How's `TemplateCategoryPicker`?

Created a separate PR: #16949